samples/doc-snippets/src/main/java/com/example/domain/TransferState.java

```diff
@@ -0,0 +1,19 @@
+package com.example.domain;
+
+public record TransferState(Transfer transfer, TransferStatus status) {
+
+  public record Transfer(String from, String to, int amount) {
+  }
+
+  public enum TransferStatus {
+    STARTED, WITHDRAW_SUCCEED, COMPLETED
```

Comment:
Should probably use past tense: `WITHDRAW_SUCCEEDED`. Or possibly `WITHDRAWN`.

Reply:
changed: https://github.com/akka/akka-sdk/pull/658/commits/40f32c72ea1fa54ccca9ab59a8e2827515e39713